python/rudder_controller.py

```diff
@@ -1,21 +1,23 @@
+# from heading_controller import HeadingController
```

Comment:
Ya I was just gonna do it in a separate PR. I was actually going to do this when refactoring the boat_controller node because it relies on the RudderController class. I can create a test file identical to the test file for the RudderController class and then push that. Then we can refactor and remove RudderController in the next PR. How about that?